clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/AddExecutor.scala

```diff
@@ -7,10 +7,12 @@ import io.circe.parser.parse
 import org.broadinstitute.clio.client.commands.{AddCommand, ClioCommand}
 import org.broadinstitute.clio.client.util.IoUtil
 import org.broadinstitute.clio.client.webclient.ClioWebClient
+import org.broadinstitute.clio.transfer.model.TransferIndex
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class AddExecutor(addCommand: AddCommand) extends Executor {
+class AddExecutor[TI <: TransferIndex](addCommand: AddCommand[TI])
```

Comment:
This seems a little odd since `TI` isn't used here; is this so you don't say `AddCommand[_]`?

Reply:
Yeah, if I use `_` then type inference breaks down for some reason (at least in IntelliJ's presentation compiler)